alpharaw/ms_data_base.py

```diff
@@ -6,7 +6,14 @@
 
 class MSData_Base:
     """
-    The base data structure for MS Data, other MSData loader inherit
+    The base data structure for MS Data, other MSData loader inherit this class.
+
+    Parameters
+    ----------
+    centroided : bool, optional
```

Comment:
does this now mean we tell the data is centroided or that centroiding should be done?

Reply:
MB centroiding is better?

Reply:
Anyway, I will change the name in other PRs, as this may affect alphadia.